manifests/0000_70_cluster-network-operator_02_rbac.yaml

```diff
@@ -1,16 +1,48 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cluster-network-operator-service-account
```

Comment:
NIT: I don't think we usually add the `-service-account` suffix to service account names, `cluster-network-operator` should suffice. This also applies to the ClusterRoleBinding object.